core/math/src/main/java/org/arakhne/afc/math/geometry/d2/ai/PathShadow2ai.java

```diff
@@ -306,7 +306,7 @@ private static void computeCrossings2(
 		if (sx0 > shadowXmax && sx1 > shadowXmax) {
 			// The line is entirely at the right of the shadow
 			if (sy1 != sy0) {
-				final double alpha = (sx1 - sx0) / (sy1 - sy0);
+				final double alpha = (double) (sx1 - sx0) / (sy1 - sy0);
```

Comment:
This change will be reverted since it causes the ECJ's "unecessary cast" warning. This warning indicates that a dynamic cast operation could be removed for making the code more efficient.